frontend/src/main/utils/UCSBArticlesUtils.js

```diff
@@ -0,0 +1,16 @@
+import { toast } from "react-toastify";
+
```

Comment:
I don't see any unit tests for UCSBArticlesUtils.js. Consider adding a file frontend/src/tests/utils/UCSBArticlesUtils.test.js to have proper test coverage (mostly for mutation coverage).